File: FragmentApp/app/src/main/java/com/gauravssnl/fragmentapp/MainActivity.java
package com.gauravssnl.fragmentapp;

import android.app.*;
import android.os.*;
import android.content.*;

public class MainActivity extends Activity 
{
    @Override
    protected void onCreate(Bundle savedInstanceState)
    {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.main);
		Intent intent = new Intent(this, SampleActivity.class);
		startActivity(intent);
		
    }
}

File: AIDE/AIDE-Samples/samples/AIDE+/KotlinApp/app/src/main/java/com/gauravssnl/app/MainActivity.kt
package com.gauravssnl.app;

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import adrt.ADRTLogCatReader
import android.view.Menu 

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        ADRTLogCatReader.onContext(this@MainActivity, "com.aide.lts")
        setContentView(R.layout.activity_main)
    }
    
    override fun onCreateOptionsMenu(menu: Menu): Boolean {
    	getMenuInflater().inflate(R.menu.activity_main_menu, menu);
    	return true;
    }
}
